packages/react-server/core/util/RequestLocalStorage.js

```diff
@@ -1,5 +1,21 @@
 var RequestLocalStorage = require('request-local-storage');
+var Promise = require('bluebird');
 
-RequestLocalStorage.patch(require('cls-q'));
+RequestLocalStorage.patch(require('cls-bluebird'));
+
+// Unfortunately `cls-bluebird` doesn't go far enough, and the maintainer
+// doesn't seem to be receptive to patches.  There's been a PR open with a
+// similar patch since 2014.  Not going to wait on that.
```

Comment:
I would include a link (on the off-chance it ever merges) and a quick explanation of why we need/example of where we use `invoke`